src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/DateTimePicker/DateTimePicker_Styles.xaml

```diff
@@ -13,8 +13,6 @@
 	<!-- DatePicker Style -->
 	<Style x:Key="DatePickerStyle"
 		   TargetType="DatePicker">
-		<ios:Setter Property="FlyoutPlacement"
-					Value="Full" />
```

Comment:
`FlyoutPlacement` was removed in https://github.com/unoplatform/uno/pull/4956. It previously existed in iOS only.